pages/admin/index.tsx

```diff
@@ -234,27 +314,30 @@ export default function Requests() {
                     Request type:{' '}
                   </Text>
                   <Text as="span" textStyle="button-regular">
-                    {requestTypeFilter}
+                    {requestTypeFilter || 'All'}
                   </Text>
                 </MenuButton>
                 <MenuList>
                   <MenuItem
                     onClick={() => {
-                      setRequestTypeFilter('All');
+                      setRequestTypeFilter(undefined);
+                      setPageNumber(0);
                     }}
                   >
                     All
                   </MenuItem>
                   <MenuItem
                     onClick={() => {
                       setRequestTypeFilter('Replacement');
+                      setPageNumber(0);
```

Comment:
can you group all the setPageNumber(0) in a useEffect hook? That it doesn't have to be repeated for every filter